apps/examples/camera/camera_main.c

```diff
@@ -169,7 +244,7 @@ static int camera_prepare(int fd, enum v4l2_buf_type type, uint32_t buf_mode, ui
 	}
 
 	if (pixformat == V4L2_PIX_FMT_UYVY) {
-		fsize = IMAGE_YUV_SIZE;
+		fsize = IMAGE_YUY_SIZE;
```

Comment:
We should consider another device, I think we can get this info from driver.

Reply:
Done.
fsize is variable set with width & height received from driver frame data